backend/uclapi/uclapi/settings.py

```diff
@@ -198,3 +198,9 @@
     FAIR_USE_POLICY = list(fp)
 
 REDIS_UCLAPI_HOST = os.environ.get("REDIS_UCLAPI_HOST")
+
+# Celery Settings
+CELERY_BROKER_URL = 'redis://' + REDIS_UCLAPI_HOST + ':6379/0'
```

Comment:
could the environment variable here contain both the `redis://` and the `:6379/0`? That seems like it will scale better

I wonder if we can give the environment variable itself a better name too? Maybe something like `CELERY_BROKER_URL`?
I think we also need an update to the .env.example